spec/services/slack_service_spec.rb

```diff
@@ -11,4 +11,15 @@
       subject.post_message('london', 'cheers!')
     end
   end
+
+  describe "Posting a message to a cheered user" do
+    let (:client) { double(:slack_client) }
+    subject { described_class.new(client) }
+
+    it "should call cheered user with expected parameters" do
+      allow(client).to receive(:users_list).with(token: 'not_a_token').and_return({"members"=>[{'name'=>'user', 'id'=>'1'}]})
+      expect(client).to receive(:chat_postMessage).with(channel: '1', text: 'you got cheered!', as_user: true)
+      subject.post_direct_message('not_a_token', ['user'], 'you got cheered!')
+    end
```

Comment:
Nice one adding a test! I think we can also test a couple more paths here. One is when the username doesn't match any in the users list that comes back from the client. And another is when there are multiple users. Maybe we can also chat about using `context` blocks for some of the setup code - want to check back in once you've looked at these new specs and maybe we can pair up on that?